src/javascript/JContent/ContentRoute/ContentRoute.jsx

```diff
@@ -32,6 +35,38 @@ export const ContentRoute = () => {
         }
     }, [dispatch, mode, viewMode, FLAT, accordionItem]);
 
+    // Captured area information is used to block delete/move/copy/cut actions on areas
+    useEffect(() => {
+        if (path && language) {
+            loadPageAndCaptureJahiaAreas(path, language, template);
+        }
+    }, [path, language, template]);
+
+    const loadPageAndCaptureJahiaAreas = (path, language, template) => {
+        const renderMode = 'editframe';
+        const encodedPath = path.replace(/[^/]/g, encodeURIComponent) + (template === '' ? '' : `.${template}`);
+        const url = `${window.contextJsParameters.contextPath}/cms/${renderMode}/default/${language}${encodedPath}.html?redirect=false`;
```

Comment:
Should we also maybe use renderUrl graphql API here vs building url manually? Not sure if we're going to encounter issues here similar to this : https://jira.jahia.org/browse/QA-15228

Reply:
yeah I think we're probably referring to the same story, I was looking at the PR you recently did here https://github.com/Jahia/jcontent/pull/1250/files?w=1 and was using renderUrl, but not sure if that was done specifically for fixing live urls. Maybe your way is more appropriate if it's for edit mode only, I'm not sure, but I just wanted to make sure the part around servername/urls are also all taken care of (when using diff sites etc).